tnb/banks.py

```diff
@@ -240,14 +240,14 @@ def post_upgrade_notice(
         }
         return self.post("/upgrade_notice", body=body)
 
-    def fetch_blocks(self):
+    def fetch_blocks(self) -> dict:
         """
         Get blocks from a Bank
         Return response as Python object
         """
         return self.fetch("/blocks")
 
-    def post_block(self, account_number, balance_key, transactions, signature):
+    def post_block(self, account_number, balance_key, transactions, signature) -> dict:
```

Comment:
Please also add typings to the parameters.

```suggestion
    def post_block(self, account_number: str, balance_key: str, transactions: list, signature: str) -> dict:
```